picard/coverart/providers/caa.py

```diff
@@ -123,8 +72,97 @@ def caa_url_fallback_list(desired_size, thumbnails):
     return urls
 
 
+class arrow_button(QtWidgets.QPushButton):
+    """Standard arrow button for CAA image type selection dialog.
+
+    Keyword Arguments:
+        label {string} -- Label to display on the button
+        command {command} -- Command to execute when the button is clicked (default: {None})
+        parent {[type]} -- Parent of the QPushButton object being created (default: {None})
+    """
+
+    ARROW_BUTTON_WIDTH = 35
+    ARROW_BUTTON_HEIGHT = 20
+
+    def __init__(self, label, command=None, parent=None ):
+        if label is None:
+            label = '.'
+        super().__init__(label, parent=parent)
+        if command is not None:
+            self.clicked.connect(command)
+        self.setFixedSize(QtCore.QSize(self.ARROW_BUTTON_WIDTH, self.ARROW_BUTTON_HEIGHT))
+
+
+class arrows_column(QtWidgets.QWidget):
+    """Standard arrow buttons column for CAA image type selection dialog.
+
+    Keyword Arguments:
+        list_add {command} -- Command to execute when the 'add' button is clicked (default: {None})
+        list_add_all {command} -- Command to execute when the 'add all' button is clicked (default: {None})
+        list_remove {command} -- Command to execute when the 'remove' button is clicked (default: {None})
+        list_remove_all {command} -- Command to execute when the 'remove all' button is clicked (default: {None})
+        reverse {bool} -- Determines whether the arrow directions should be reversed (default: {False})
+        parent {[type]} -- Parent of the QWidget object being created (default: {None})
+    """
+
+    def __init__(self, list_add=None, list_add_all=None, list_remove=None, list_remove_all=None, reverse=False, parent=None):
+        _spacer_item = QtWidgets.QSpacerItem(20, 20, QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Expanding)
+        _arrows_layout = QtWidgets.QVBoxLayout()
```

Comment:
Since `_arrows_layout` and `_space_item` are local to this method, no need to prefix with `_` (unlike a "private" property, ie. `self._arrows_layout`)

Reply:
Fixed.  Looks much better.